src/network/frame_decoder.rs

```diff
@@ -7,10 +7,10 @@ use crate::network::models::{
 use crate::network::{CacheInfo, ObjectAttribute, VersionTriplet};
 use crate::parser::ReplayBody;
 use bitter::BitGet;
-use failure::Error;
 use fnv::FnvHashMap;
 use std::collections::HashMap;
 use std::ops::Deref;
+use core::convert::From;
```

Comment:
No need to import `From`

Reply:
Must be a bug with CLion's Rust extension then! It fails to detect `usize::from(object_id)` and my need to get rid of red lines with it's suggestions was unwarranted.  It was doing the same for `use core::iter::Iterator;` when I changed Vec<Frame> to &[Frame].